src/lib/features/authenticate/model/sign-in-snackbar.ts

```diff
@@ -1,5 +1,8 @@
 import { snackbar } from '$lib/shared/ui/snackbar';
 import { supabaseClient } from '$lib/shared/api';
+import { likesStore, addCloudLike } from '$lib/features/like-episode';
+import { listeningHistory, addToCloudListeningHistory } from '$lib/features/listening-history';
```

Comment:
issue: FSD, cross-imports of features

Reply:
Same as https://github.com/aabounegm/cast/pull/241#discussion_r867538147